### ISSUE TITLE
Change order of the command line option when creating an account

### ISSUE THREAD

**Issue by majecty** [CodeChain-io/codechain#482]
Currently below line throw error
```sh
codechain account create --keys-path keys/key0

error: Found argument '--keys-path' which wasn't expected, or isn't valid in this context
```

Below line is ok
```sh
codechain account  --keys-path keys/key0   create
```

**Comment by joojis** [comment]
Patch: https://github.com/CodeChain-io/codechain/pull/598

**Comment by kseo** [comment]
Landed in 725716f98a6e9bf79b0dafeface4c8620f8b7a91
